src/sidebar/search-client.js

```diff
@@ -90,6 +91,11 @@ export default class SearchClient extends TinyEmitter {
         }
 
         const chunk = results.rows.concat(results.replies || []);
+        if (!self._resultCount) {
```

Comment:
If you make the change I suggested above this would need to be changed to `if (self._resultCount === null) { ... }`.